packaging/windows.cmake

```diff
@@ -0,0 +1,56 @@
+set(CPACK_GENERATOR "NSIS")
+find_package(cx_freeze 5.0 REQUIRED)
+
+configure_file(${CMAKE_CURRENT_LIST_DIR}/setup_win32.py.in setup.py @ONLY)
+add_custom_target(build_bundle)
+add_dependencies(packaging build_bundle)
+add_dependencies(build_bundle cura-binary-data)
+
+# TODO: Find a variable which holds the needed "win32"/"win64"
+#       There is a CPACK* variable which holds this variable, but it doesn't exist at this moment here...
+if(${BUILD_OS_WIN32})
+    set(NSIS_SCRIPT_COPY_PATH "${CMAKE_BINARY_DIR}/_CPack_Packages/win32/${CPACK_GENERATOR}")
```

Comment:
Rather than try and divine the win32/win64 here, why not set CPACK_PACKAGE_DIRECTORY to something known? Doing 32-bit/64-bit builds from the same build directory is not supported anyway.